.github/workflows/cla.yml

```diff
@@ -0,0 +1,26 @@
+name: 'CLA Assistant'
+on:
+  issue_comment:
+    types: [created]
+  pull_request_target:
+    types: [opened, closed, synchronize]
+
+jobs:
+  CLAssistant:
+    runs-on: ubuntu-latest
+    steps:
+      - name: 'CLA Assistant'
+        if: (github.event.comment.body == 'recheckcla' || github.event.comment.body == 'I have read the CLA Document and I hereby sign the CLA') || github.event_name == 'pull_request_target'
+        # Alpha Release
+        uses: gnosis/cla-github-action@master
+        # GitHub token, automatically provided to the action
+        # (No need to define this secret in the repo settings)
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          path-to-signatures: 'signatures/version1/cla.json'
+          path-to-cla-document: 'https://gnosis-safe.io/cla/'
+          branch: 'cla-signatures'
```

Comment:
From the docs it seems that this is going to store the signatures in a branch named `cla-signatures` Under a file named `signatures/version1/cla.json`.

It's a bit weird to have a separate branch to just keep track of the signatures but I guess that's how the action works and why it needs and unprotected branch. But an unprotected branch can be also a trigger for force pushes who change the history of the signatures 🤔

Reply:
I guess the assumption is that we would never use `cla-assitant` as a base branch against which we would open a PR